.github/workflows/ci.yml

```diff
@@ -39,9 +39,10 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-build-${{ env.cache-name }}-${{ matrix.node-version }}-
     - run: npm install --no-audit
-    - run: npm run format-check
```

Comment:
Can you open a new PR that reverts this file in order to let someone work on fixing the underlying problem? My preference is for us not to go too long without these.

Reply:
....just ignore me